integration-tests/src/main/java/org/apache/druid/testing/guice/DruidTestModule.java

```diff
@@ -85,6 +85,8 @@ public HttpClient getHttpClient(
   @ManageLifecycle
   public ServiceEmitter getServiceEmitter(Supplier<LoggingEmitterConfig> config, ObjectMapper jsonMapper)
```

Comment:
Can both the params also be removed now?

Reply:
Updated.